Classes/Typo3PageIndexer.php

```diff
@@ -315,7 +315,7 @@ protected function getPageDocument()
 
         // access
         $access = (string)$this->pageAccessRootline;
-        if (trim($access) !== "") {
+        if (trim($access) !== '') {
```

Comment:
I tried to avoid functional changes. csfixer replaces double quotes with single quotes.